src/dmd/frontend.h

```diff
@@ -2024,8 +2024,8 @@ class LinkDeclaration final : public AttribDeclaration
 {
 public:
     LINK linkage;
-    static LinkDeclaration* create(LINK p, Array<Dsymbol* >* decl);
-    LinkDeclaration* syntaxCopy(Dsymbol* s);
+    static LinkDeclaration* create(const Loc& loc, LINK p, Array<Dsymbol* >* decl);
+    Dsymbol* syntaxCopy(Dsymbol* s);
```

Comment:
This is the reason why the tests are failing. It seems that the return of `syntaxCopy` was not modified in the D file. Why is it modified in the header files?